### PR TITLE
chore: add config for assert schema timeout

### PR DESCRIPTION
**Pull request by jzaralim**
### Description 
~changed `ksql.assert.topic.default.timeout.ms` to `ksql.assert.default.timeout.ms ` because it covers schemas as well~

Adds a separate config for schema timeouts, `ksql.assert.schema.default.timeout.ms`

### Testing done 
should pass all tests

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

**Comment by jnh5y** [comment]
Just to argue (quickly and weakly) about the premise, is there any chance that folks would want to manage timeouts for `assert topic` separately form `assert schema`?  

Admittedly, that's decently fine-grained / fussy, but I could imagine topic creation and schema registration taking different amounts of time.

File: ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java [hunk 1]
```diff
@@ -661,11 +661,11 @@ public class KsqlConfig extends AbstractConfig {
   private static final boolean KSQL_ENDPOINT_MIGRATE_QUERY_DEFAULT = true;
   private static final String KSQL_ENDPOINT_MIGRATE_QUERY_DOC
       = "Migrates the /query endpoint to use the same handler as /query-stream.";
-  public static final String KSQL_ASSERT_TOPIC_DEFAULT_TIMEOUT_MS
-      = "ksql.assert.topic.default.timeout.ms";
-  private static final int KSQL_ASSERT_TOPIC_DEFAULT_TIMEOUT_MS_DEFAULT = 1000;
-  private static final String KSQL_ASSERT_TOPIC_DEFAULT_TIMEOUT_MS_DOC
-      = "The default timeout for ASSERT TOPIC statements if not timeout is specified "
+  public static final String KSQL_ASSERT_DEFAULT_TIMEOUT_MS
```
Comment (jnh5y) [inline ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java:664]: For those playing the home game, this introduced in https://github.com/confluentinc/ksql/commit/cd5254f9214fe0faadb98d505664f866805aedf0 which has not been released, so nothing public and published is changing.  (I had to check for my own sanity.:))

**Comment by jzaralim** [comment]
That's a fair point. It's not a huge difference anyway, so lets give them the options

**Review by jnh5y — APPROVED** [review]
Awesome!

**Pull request merged**